### ISSUE TITLE
AMA Edit | Confirmation

### ISSUE THREAD

**Issue by suzchap** [department-of-veterans-affairs/caseflow#7273]
## Story
For any AMA forms, when a user needs to edit issues, they will click the "Edit in Caseflow" button in VBMS to return to the "Add issues landing page" where they will be able to add new issues and/or remove issues added previously. There are a few "are you sure?" type modals to help reduce errors. [Entire flow via Mural](https://app.mural.co/t/workqueue2001/m/workqueue2001/1537795938997/f5834747503f5a248340b9f3426fbac652812400)

This ticket is focused on the Edit Confirmation page that you get after adding or removing at least one issue and clicking "Save" from the Add Issue Landing page.

## Technical notes
- The 2 sentences under the page title should be styled as "Lead Paragraph"

## Acceptance criteria
- From the Add Issues Landing page, after you edit issues (either add or remove), the "Save" button becomes active. When clicked, you are taken to the Edit Confirmation page.
- The confirmation page includes the same information as previous iterations with the exception of a revised sentence about VBMS (see copy section and mock).

## Notes
Out of scope:
- The overall content / design for all the variations of the confirmation page will be updated in a future ticket so don't worry about minor details too much in this iteration.

## Copy
(All copy same as prior version except sentence about VBMS should be changed to...)
Go to VBMS claim details  and reload to view updates.

## Mocks
<img width="691" alt="screen shot 2018-10-05 at 10 15 48 am" src="https://user-images.githubusercontent.com/11219417/46540549-ac338d80-c887-11e8-8eed-c9e7efd54efe.png">

## How to Validate
(no need, Shane's got this :muscle:)

**Comment by shanear** [comment]
The copy here does not reflect that the operation may not happen immediately.

**Comment by suzchap** [comment]
@shanear I was planning to update this version of the confirmation page in a following ticket to be done after #7308

**Comment by shanear** [comment]
cool then we can close this one.
